### PR TITLE
Release

### PR DESCRIPTION
**Pull request by Lennart99**
We have created the basic framework for the module scheduler, including a easy way to initialize tasks for modules and a way to wait for the communication bus.
We also created a example for future users, to make sure it is clear how this module should be used
Lastly we intergrated the moving-platform module into the scheduler.

### CODE REVIEW

**Review by florianhumblot — CHANGES_REQUESTED** [review]
Please add doxygen documentation to the `alloc_c` and `arena_alloc_c` classes.
Also pick one doxygen style for the module, don't mix the `/**/` style with `//`. 

Let me know when that's done so I can re-review this PR

**Comment by Lennart99** [comment]
> Please add doxygen documentation to the `alloc_c` and `arena_alloc_c` classes.
> Also pick one doxygen style for the module, don't mix the `/**/` style with `//`.
> 
> Let me know when that's done so I can re-review this PR

fixed in https://github.com/R2D2-2019/module_scheduler/pull/3/commits/7f88c053925850147af1f0f8df3a347fe4c61c8c

**Review by florianhumblot — APPROVED** [review]

File: Makefile.due [hunk 1]
```diff
@@ -15,6 +15,10 @@ CONSOLE_BAUDRATE  ?= 2400
 STACK_SIZE        ?= 81920
 HEAP              ?= BMPTK
 
+# add rtos
+RTOS              ?= ../../../libraries/rtos
```
Comment (PatrickDekker98) [inline Makefile.due:19]: Why is RTOS included in makefile.due and not in makefile.link?

File: README.md [hunk 2]
```diff
@@ -0,0 +1 @@
+# robos
```
Comment (PatrickDekker98) [inline README.md:1]: Empty readme, be sure to add a readme in the future ;)

File: Makefile.due (hunk 1 above)
Comment (PatrickDekker98) [reply Makefile.due:19]: I just realized rtos does not work on native

**Review by PatrickDekker98 — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (Lennart99) [reply README.md:1]: we use the wiki for that, right? :)

**Pull request merged**